src/test_driver/linux-cirque/helper/CHIPTestBase.py

```diff
@@ -130,34 +130,33 @@ def sequenceMatch(self, string, patterns):
 
     def connect_to_thread_network(self):
         self.logger.info("Running commands to form Thread network")
-        time.sleep(3)  # Avoid sending commands at very beginning.
+        for device in self.non_ap_devices:
+            self.wait_for_device_output(device['id'], "Border router agent started.", 5)
         otInitCommands = [
             "ot-ctl thread stop",
             "ot-ctl ifconfig down",
-            "ot-ctl dataset init new",
-            "ot-ctl dataset panid 0x1234",
-            "ot-ctl dataset networkname OpenThread",
-            "ot-ctl dataset channel 13",
-            "ot-ctl dataset extpanid dead00beef00cafe",
-            "ot-ctl dataset meshlocalprefix \"fd01:2345:6789:0abc::\"",
-            "ot-ctl dataset masterkey 00112233445566778899aabbccddeeff",
-            "ot-ctl dataset commit active",
-            "ot-ctl dataset active",  # This will emit an output of dataset in flask.log
+            "ot-ctl dataset set active 0e080000000000010000000300000d35060004001fffe00208dead00beef00cafe0708fd01234567890abc051000112233445566778899aabbccddeeff030a4f70656e546872656164010212340410ad463152f9622c7297ec6c6c543a63e70c0302a0ff",
```

Comment:
should we in any way document what this dataset contains? or is it supposed to be completely opaque?

Reply:
Added a comment on this function to clearify that this function is used to create a thread network that the user does not care about the detail of the network, and the dataset is used to replace the default dataset created by openthread.